packages/webpack-cli/lib/utils/cli-executer.js

```diff
@@ -1,64 +0,0 @@
-const { MultiSelect, Input } = require('enquirer');
-const { cyan } = require('colorette');
-const logger = require('./logger');
-const cliArgs = require('./cli-flags').core;
-
-const prompter = async () => {
-    const args = [];
-
-    const typePrompt = new MultiSelect({
```

Comment:
We can use autocomplete prompt here to avoid long list.

Reply:
We can't because user can't use shell, not each shell support autocomplete

Reply:
There is also a AutoComplete prompt in enquirer. I talking about it